backend/app/adapter/gqlapi/resolver/error.go

```diff
@@ -77,16 +78,20 @@ func (e ErrNotHuman) Error() string {
 }
 
 // ErrInvalidLongLink signifies that the provided long link has incorrect format.
-type ErrInvalidLongLink string
+type ErrInvalidLongLink struct {
+	longLink  string
+	violation string
+}
 
 var _ GraphQlError = (*ErrInvalidLongLink)(nil)
 
 // Extensions keeps structured error metadata so that the clients can gracefully
 // handle the error.
 func (e ErrInvalidLongLink) Extensions() map[string]interface{} {
 	return map[string]interface{}{
-		"code":     ErrCodeInvalidLongLink,
-		"longLink": string(e),
+		"code":      ErrCodeInvalidLongLink,
+		"longLink":  string(e.longLink),
```

Comment:
```suggestion
		"longLink":  e.longLink,
```

Reply:
I think they are already strings based on line 82 to 83.

Reply:
Done